pkg/interface/src/logic/subscription/global.ts

```diff
@@ -43,12 +43,25 @@ export default class GlobalSubscription extends BaseSubscription<StoreState> {
     groups: []
   };
   start() {
-    this.subscribe('/all', 'invite-store');
+    this.scry('/all', 'invite-store');
+    this.subscribe('/updates', 'invite-store');
+
+    this.scry('/groups', 'group-store');
     this.subscribe('/groups', 'group-store');
```

Comment:
Well, you'll want /updates not /groups here. You're getting /groups from the scry, why make two requests... that is, if group-store has an updates subscription.

Reply:
It does not currently.